adr/0014-home-assistant-supervised.md

```diff
@@ -27,6 +27,7 @@ Docker CE (Community Edition) is the only supported containerization method for
 - NetworkManager >= 1.14.6
 - AppArmor == 2.13.x (built into the kernel)
 - Debian Linux Debian 10 aka Buster (no derivatives)
+- Home Assistant OS-Agent (close as possible to latest release)
```

Comment:
```suggestion
- [Home Assistant OS-Agent](https://github.com/home-assistant/os-agent) (Only the [latest release](https://github.com/home-assistant/os-agent/releases/latest) is supported)
```

Reply:
Right, we don't need force so strict the requirements but at least it's still clear what is need to do

Reply:
I mean, the suggestion is fine for me :)